jenkins-integration/beaker/install/pe/98_sync_codedir.rb

```diff
@@ -1,27 +0,0 @@
-test_name 'Sync code-staging to code via File Sync'
```

Comment:
This file is still used by PE gatling tests elsewhere in the repo. Please don't remove it.